acceptance/api/v1/suite_test.go

```diff
@@ -45,16 +47,11 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	}
 
 	fmt.Printf("Compiling Epinio on node %d\n", config.GinkgoConfig.ParallelNode)
-	testenv.BuildEpinio()
 
 	testenv.CreateRegistrySecret()
 
-	epinioBinary := testenv.EpinioBinaryPath()
-	err := testenv.EnsureEpinio(epinioBinary)
-	Expect(err).ToNot(HaveOccurred(), "installing epinio")
-
-	out, err := testenv.PatchEpinio()
```

Comment:
Search for "PatchEpinio" in this diff. You remove it here but you keep it in other cases. Shouldn't the Make target take care of everything, including patching? Maybe we can also remove this function, unless the installation suite still needs it.

Reply:
Right. Removed now in other places